packages/redux/src/contents/selectors.ts

```diff
@@ -282,5 +317,5 @@ export const getSEOMetadataResult = (
  *
  * @returns Content normalized.
  */
-export const getContent = (state: StoreState, hash: string) =>
-  getEntityById(state, 'contents', hash);
+export const getContent = <T>(state: StoreState, hash: string) =>
+  getEntityById(state, 'contents', hash) as unknown as ContentEntry<T>;
```

Comment:
`getEntityById` will return undefined if `hash` is not found, so the cast to unknown should not be made. Can you please remove it and reapply the case to `ContentEntry[]` in the getContents selector?